src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -296,11 +301,15 @@
       <Link>Common\Interop\OSX\Interop.Libraries.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetsFreeBSD)' == 'true' OR '$(TargetsUnknownUnix)' == 'true' ">
+  <ItemGroup Condition=" ('$(TargetsFreeBSD)' == 'true' OR '$(TargetsUnknownUnix)' == 'true') AND '$(IsPartialFacadeAssembly)'!='true'">
```

Comment:
Do we need the second part? Doesn't being net46 or net461 (and therefore a partial facade per above) automatically mean TargetsWindows and therefore neither of the first two conditions? @weshaggard